translations.yml

```diff
@@ -349,9 +349,9 @@ parts:
       key: "txt.help_center_copenhagen_theme.show_brand_name_label"
       title: "Label for the show brand name setting"
       screenshot: "https://drive.google.com/file/d/16zQao3zk3uRScIUpkyc6bXAWwqHh3Zh-/view?usp=sharing"
-      value: "Show Help Center name"
+      value: "Show brand name"
```

Comment:
I don't think we can update these strings directly since they were already submitted for translation (see [here](https://zendesk.atlassian.net/wiki/spaces/globalization/pages/742069700/Edit+Translations)). Can you please double check it with @zendesk/localization ? We usually mark strings as obsolete and add new ones instead.

Reply:
I don't see a yml here anymore? but indeed please obsolete and create a new key for those types of changes :)